src/parser/parser_driver.cpp

```diff
@@ -1320,52 +1320,72 @@ void ParserDriver::defineGrammar()
 				return output;
 			}
 		)
-		.production("for_expression", "OF", "string_set", [&](auto&& args) -> Value {
+		//.production("for_expression", "OF", "string_set", [&](auto&& args) -> Value {
```

Comment:
Nit: What is the reason behind commented out code? Do we want to leave it in?

Reply:
The commented out code is there just for the sake of completeness. Unfortunately, such grammar is not LALR(1) grammar because it contains reduce/reduce conflicts. I kept it there because what I plan to do is to release yaramod 4.x.x and put into README that yaramod 4.x.x is switching to YARA-X compatibility mode and that 3.x.x should be used to maintain the old YARA compatibility. I want to end this message with basically stating that yaramod is about the be completely deprecated by new YARA-X and that further support for 4.x.x is not planned. So 4.x.x is just a version that is about to be phased out of existence and this old code doesn't really bother me.

Reply:
Cool, thanks for the explanation :pray: